data/theme/cinnamon.css

```diff
@@ -76,6 +76,13 @@ StScrollBar StButton#vhandle:hover {
     font-weight: normal;
     text-align: center;
 }
+.ripple-pointer-location {
+  width: 50px;
+  height: 50px;
+  border-radius: 25px 25px 25px 25px;
```

Comment:
@JosephMcc This PR failed due to this being a redundant longhand value.
Changing it to `border-radius: 25px;` should fix it.